frontend/public/scss/product-page/product-details.scss

```diff
@@ -325,7 +321,7 @@ body.new-design {
 
     .acheiving-text {
```

Comment:
I know it doesn't matter how this is spelled, but after we make the text changes, it will prob confuse folks to have this as the class name.